client/core/trade.go

```diff
@@ -1592,7 +1732,6 @@ func (c *Core) tick(t *trackedTrade) (assetMap, error) {
 			redeems = append(redeems, match)
 			received += match.Quantity
 			quoteReceived += calc.BaseToQuote(match.Rate, match.Quantity)
-			return nil
```

Comment:
unintended removal I think

Reply:
Perhaps you wanted to always get to `checkSwapFeeConfirms`?  Could move the call to `checkSwapFeeConfirms` to happen between `isSwappable` and `isRedeemable` instead of below.

Reply:
I don't know now. I don't think it need be removed but I did move up the `checkSwapFeeConfirms`